docker_kernel/magics/tag.py

```diff
@@ -35,24 +37,25 @@ def VALID_OPTIONS() -> dict[str, FlagDict]:
         }
     
     def _execute_magic(self) -> None:
-        target = self._args[0]
+        """
+        usage:
+        %tag source_image[:tag] target_image[:tag]
+        """
+        source_image = self._args[0]
+        target_image = self._args[1]
         try:
-            name, tag = target.split(":")
+            name, tag = target_image.split(":")
```

Comment:
Maybe instead of checking this condition here, we could use the `ARGS_RULES()` to specify lambda functions that check all of this. This can be done later though!

Reply:
I will do this later.